partner_communication_tag/models/partner_communication_tag.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2018 Rémy Taymans <remytaymans@gmail.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+
+from odoo import models, fields
+
+
+class PartnerCommunicationTag(models.Model):
+    _name = 'res.partner.communication.tag'
+    _description = "Partner Communication Tag"
+    _order = 'name'
+
+    name = fields.Char("Name", required=True)
+    description = fields.Text("Description")
```

Comment:
use `translate=True` to translate field values

Reply:
should this be added for both `name` and `description` fields?